LunaMary/src/Ask.java

```diff
@@ -0,0 +1,42 @@
+import java.util.List;
+import java.util.Scanner;
+
+
+class Ask {
```

Comment:
Maybe you can change the name to: Question instead of Ask. I think is more self explanatory.